### DISCUSSION TITLE
Suddenly stopped working

### DISCUSSION THREAD

**Discussion by acrylicus** [Taitava/obsidian-shellcommands#441] [Debugging and testing]
### Obsidian debug information

Latest obsidian Version 1.7.7 
MacOS 15.2 (24C98) Sequoia
Shell Commands 0.23.0

### What shell are you using?

Zsh

### Please describe the problem

Everything worked up until recently, when suddenly it stopped.

My usecase is very simple, I have a daily scipt that I want to execute simply called "daily.sh". The script is fine, when I run it from the shell normally it works as expected. 

I get no console errors, and I konw the script doesn't run as I put a "touch" at the very top which doesnt apply to my selected file.

The only update I can think of is a recent OhMyZsh update... 

My config is very simple
<img width="826" alt="image" src="https://github.com/user-attachments/assets/d93d39ff-9abb-4101-9517-d531e0d93649" />

**Comment by Taitava** [comment]
Hi! This sounds strange. 

Does it work if you downgrade the SC plugin to version 0.22.0? I'm not aware of any issues in the current version 0.23.0, but just to try something.

If SC 0.22.0 has the same problem, then perhaps try to downgrade Obsidian to 0.7.5. That is [the latest version of Obsidian I have run my SC test vault on](https://mydomain.com/index.html). However, I have used SC 0.23.0 with Obsidian 1.8.0 (I guess it's still early access?) on Windows 10 and the few shell commands I have used today have executed ok. I can't remember if I have used SC on Obsidian 1.7.7 particularly.

- If none of these downgrades help, we'll need to think about something else.
- If any of them helped, then I can start further digging into what has changed and how to fix it. 🙂
